tests/test_globalization.py

```diff
@@ -91,161 +121,82 @@ def get_next_characters(all_characters, count=50):
             u'ꆩꆪꆫꆬꆭꆮꆯꆰꆱꆲꆳꆴꆵꆶꆷꆸꆹꆺꆻꆼꆽꆾꆿꇀꇁꇂꇃꇄꇅꇆꇇꇈꇉꇊꇋꇌꇍꇎꇏꇐꇑꇒꇓꇔꇕꇖꇗꇘꇙꇚꇛꇜꇝꇞꇟꇠꇡꇢꇣꇤꇥꇦꇧꇨꇩꇪꇫꇬꇭꇮꇯꇰꇱꇲꇳꇴꇵꇶꇷꇸꇹꇺꇻꇼ'
             u'ꇽꇾꇿꈀꈁꈂꈃꈄꈅꈆꈇꈈꈉ'
         ),
-        'zang': (
+        # zang
+        (
             u'ꃙꃚꃛꃜꃝꃞꃟꃠꃡꃢꃣꃤꃥꃦꃧꃨꃩꃪꃫꃬꃭꃮꃯꃰꃱꃲꃳꃴꃵꃶꃷꃸꃹꃺꃻꃼꃽꃾꃿꄀꄁꄂꄃꄄꄅꄆꄇꄈꄉꄊꄋꄌꄍꄎꄏꄐꄑꄒꄓꄔꄕꄖꄗꄘꄙꄚꄛꄜꄝꄞꄟꄠꄡꄢꄣꄤꄥꄦꄧꄨꄩꄪꄫꄬꄭ'
             u'ꄮꄯꄰꄱꄲꄳꄴꄵꄶꄷꄸꄹꄺꄻꄼꄽꄾꄿꅀꅁꅂꅃꅄꅅꅆꅇꅈꅉꅊꅋꅌꅍꅎꅏꅐꅑꅒꅓꅔꅕꅖꅗꅘꅙꅚꅛꅜꅝꅞꅟꅠꅡꅢꅣꅤꅥꅦꅧꅨꅩꅪꅫꅬꅭꅮꅯꅰꅱꅲꅳꅴꅵꅶꅷꅸꅹꅺꅻꅼꅽꅾꅿꆀꆁꆂ'
             u'ꆃꆄꆅꆆꆇꆈꆉꆊꆋꆌꆍꆎꆏꆐꆑꆒꆓꆔꆕꆖꆗꆘꆙꆚꆛꆜꆝꆞꆟꆠꆡꆢꆣꆤꆥꆦꆧꆨꆩꆪꆫꆬꆭꆮꆯꆰꆱꆲꆳꆴꆵꆶꆷꆸꆹꆺꆻꆼꆽꆾꆿꇀꇁꇂꇃꇄꇅꇆꇇꇈꇉꇊꇋꇌꇍꇎꇏꇐꇑꇒꇓꇔꇕꇖꇗ'
             u'ꇘꇙꇚꇛꇜꇝꇞꇟꇠꇡꇢꇣꇤꇥꇦꇧꇨꇩꇪꇫꇬꇭꇮꇯꇰꇱꇲꇳꇴꇵꇶꇷꇸꇹꇺꇻꇼꇽꇾꇿꈀꈁꈂꈃꈄꈅꈆꈇꈈꈉꈊꈋꈌꈍꈎꈏꈐꈑꈒꈓꈔꈕꈖꈗꈘꈙꈚꈛꈜꈝꈞꈟꈠꈡꈢꈣꈤꈥꈦꈧꈨꈩꈪꈫꈬ'
             u'ꈭꈮꈯꈰꈱꈲꈳꈴꈵꈶꈷꈸꈹꈺꈻꈼꈽꈾꈿꉀꉁꉂꉃꉄꉅꉆꉇꉈꉉꉊꉋꉌꉍꉎꉏꉐꉑꉒꉓꉔꉕꉖꉗꉘꉙꉚꉛꉜꉝꉞꉟꉠꉡꉢꉣꉤ'
         )
-    }
+    ]
 
-
-class TestGlobalizationResultSet(GlobalizationTests):
-    def test_charset_double(self):
-        self.run_charset_validation('double')
-
-    def test_charset_four(self):
-        self.run_charset_validation('four')
-
-    def test_charset_mongolian(self):
-        self.run_charset_validation('mongolian')
-
-    def test_charset_uyghur(self):
-        self.run_charset_validation('uyghur')
-
-    def test_charset_tibetian(self):
-        self.run_charset_validation('tibetian')
-
-    def test_charset_yi(self):
-        self.run_charset_validation('yi')
-
-    def test_charset_zang(self):
-        self.run_charset_validation('zang')
-
-    def run_charset_validation(self, charset):
+    @pytest.mark.parametrize("charset", test_charset_data)
```

Comment:
Using pytest, you no longer get integrated tests in vscode. Is there a reason why we're change this?

Reply:
I don't believe that's true about Pytest anymore ([source](https://code.visualstudio.com/docs/python/testing)).

Pytest provides a lot of benefits over unittest, especially in readability of tests. In this particular instance we're using parametrized tests, enabling us to define one test method for multiple tests. This allowed us to remove redundant code.

Reply:
pytest doesn't work well with parameterized tests. Gene, the engineer before you, ported all pytests to be unittest tests due to this issue. I'd prefer we just leave them as is for now.

Reply:
That's very interesting—truthfully I've already ported parametrized tests for 4 other test files (interactive, non_interactive, mssqlclient, multiline). Also looks like our original merge uses parametrized tests for ctes and parseutils.

I'll defer to your judgement if you still prefer to remove this, but in my experience it works great.

Reply:
As long as it easy to debug a particular language, without having to others, I'm okay with it I guess. What's the command line to run pytest with a specific parameter set?

Reply:
Yes there's still the ability to log which language fails. I'm making a new commit to accommodate this.

I'm not positive on how to call a test from the command line with just a single parameter. [This StackOverflow post](https://stackoverflow.com/questions/44943470/parametrize-and-running-a-single-test-in-pytest) may show it's possible. I usually comment out the other parameters and call `pytest test_mod.py::TestClass::test_method`.

Reply:
So that's what I want to avoid - having to comment out tests to run a test.

Reply:
I've validated you can test for a single parameter using the `-k` flag, followed by a string which matches the parameter to test.